Sources/BraintreePayPal/BTPayPalLineItem.swift

```diff
@@ -54,11 +88,11 @@ import Foundation
         self.kind = kind
     }
     
-    // MARK: - Internal Methods
+    // MARK: - Public Methods
 
     /// Returns the line item in a dictionary.
     /// - Returns: A dictionary with the line item information formatted for a request.
-    func requestParameters() -> [String: String] {
+    public func requestParameters() -> [String: String] {
```

Comment:
Why does this need to be public now? Doesn't seem to be used publicly in this PR